hack/test-cmd.sh

```diff
@@ -55,27 +58,45 @@ unset KUBECONFIG
 
 export ALLOWED_REGISTRIES='[{"domainName":"172.30.30.30:5000"},{"domainName":"myregistry.com"},{"domainName":"registry.centos.org"},{"domainName":"docker.io"},{"domainName":"gcr.io"},{"domainName":"quay.io"},{"domainName":"*.redhat.com"},{"domainName":"*.docker.io"},{"domainName":"registry.redhat.io"}]'
 
-os::start::configure_server
+eval $( ./hack/install-etcd.sh | tail -n1 )
+LOCALUP_ROOT=$(mktemp -d) localup::init_master
+
+export ADMIN_KUBECONFIG="${LOCALUP_CONFIG}/admin.kubeconfig"
 
 os::test::junit::declare_suite_start "cmd/version"
-os::cmd::expect_success_and_not_text "KUBECONFIG='${MASTER_CONFIG_DIR}/admin.kubeconfig' oc version" "did you specify the right host or port"
+os::cmd::expect_success_and_not_text "KUBECONFIG='${LOCALUP_CONFIG}/admin.kubeconfig' oc version" "did you specify the right host or port"
 os::cmd::expect_success_and_not_text "KUBECONFIG='' oc version" "Missing or incomplete configuration info"
 os::test::junit::declare_suite_end
 
-os::start::master
-
-os::start::registry
-
 export HOME="${FAKE_HOME_DIR}"
 mkdir -p "${HOME}/.kube"
-cp "${MASTER_CONFIG_DIR}/admin.kubeconfig" "${HOME}/.kube/non-default-config"
+cp "${LOCALUP_CONFIG}/admin.kubeconfig" "${HOME}/.kube/non-default-config"
 export KUBECONFIG="${HOME}/.kube/non-default-config"
 export KUBERNETES_MASTER="${API_SCHEME}://${API_HOST}:${API_PORT}"
 
 # Store starting cluster RBAC to allow it to be restored during tests that mutate the global policy
 BASE_RBAC_DATA="$( mktemp "${BASETMPDIR}/base_rbac_data_XXXXX" )"
 export BASE_RBAC_DATA
+
 oc get clusterroles.rbac,clusterrolebindings.rbac -o yaml | grep -v resourceVersion > "${BASE_RBAC_DATA}"
+# these should probably be set up in localup::init_master
```

Comment:
Why do they have to be set at all?>

Reply:
Because I needed something that works first